src/GUI/ReverseEngineer20/ReverseEngineer20.csproj

```diff
@@ -141,15 +141,24 @@
       <HintPath>..\lib\Microsoft.SqlServer.Types.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="MySqlConnector, Version=0.49.2.0, Culture=neutral, PublicKeyToken=d33d3e53aa5f8c92, processorArchitecture=MSIL">
+      <HintPath>..\packages\MySqlConnector.0.49.2\lib\net471\MySqlConnector.dll</HintPath>
+    </Reference>
+    <Reference Include="Newtonsoft.Json, Version=11.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
```

Comment:
Is this due to MySqlConnector?

Reply:
It is from Pomelo.EntityFrameworkCore.MySql depending on  MySqlConnector (>= 0.49.2) and Pomelo.JsonObject which in turn depends on  Newtonsoft.Json (>= 11.0.2)